<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admcontrol</class>
 <widget class="QDialog" name="admcontrol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>625</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>admcontrol</string>
  </property>
  <widget class="QTabWidget" name="addbook">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>添加书籍</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>120</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>391</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="name_1">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,0);
	border:none;
	border-bottom:1px solid black;}</string>
        </property>
        <property name="placeholderText">
         <string>书名：</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="author">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,0);
	border:none;
	border-bottom:1px solid black;}</string>
        </property>
        <property name="placeholderText">
         <string>作者：</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="publisher">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,0);
	border:none;
	border-bottom:1px solid black;}</string>
        </property>
        <property name="placeholderText">
         <string>出版社：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>180</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>18</height>
      </size>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>300</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>250</y>
       <width>60</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="resultLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>请选择类别：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>删除书籍（列表式）</string>
    </attribute>
    <widget class="QLineEdit" name="name_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,0);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string>书名：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>621</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>查找书籍</string>
    </attribute>
    <widget class="QLineEdit" name="name_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,0);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string>书名：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>130</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>读者状态控制</string>
    </attribute>
    <widget class="QLineEdit" name="readername_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgba(255,255,255,0);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string>解禁读者名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="readername">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgba(255,255,255,0);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string>封禁读者名：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>110</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>237</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>180</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>书籍数据</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>name_1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>author</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>publisher</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
